app/controllers/applications_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationsController < ApplicationController
-  before_action :find_project, except: [:index]
+  before_action :find_project, only: [:show, :create, :destroy, :new]
```

Comment:
NOTE: changed this so that it doesnt break for the accept feature!

Reply:
what, is that it??? :D

Reply:
Good spot @Haumer ! @E-F-J  probably easiest to add the 'accept' feature changes after your PR as only a few lines!

Reply:
@E-F-J you where calling a before action for all actions except the index. this mean if an action does not have access to a project_id it will break so it safer to do the inverse and call `only` for the actions that need a project! (the accept action doesnt need a project 😄)

Reply:
makes sense! thanks a lot, have a great week off!